src/components/Addbook.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const Addbook = () => (
+  <div>
+    <h3>ADD NEW BOOK</h3>
+    <div className="addbook">
+      <input type="text" name="book-name" required placeholder="Book Title" />
+      <select name="category">
+        <option value="Action">Sports</option>
+        <option value="Horror">Weather</option>
+        <option value="Comedy">Comedy</option>
+        <option value="Magic">Science Fiction</option>
+      </select>
+      <button type="button">ADD BOOK</button>
+    </div>
+  </div>
+);
+
+export default Addbook;
```

Comment:
- [ ] Even though you are not supposed to implement the add book button functionalities. however,I would suggest to implement what you did  above in the redux folder and book.js and render that in the component. As you see your redux folder has empty file.